agent/connect/uri_service.go

```diff
@@ -52,3 +53,33 @@ func (id SpiffeIDService) uriPath() string {
 	}
 	return path
 }
+
+// SpiffeIDIdentity is the structure to represent the SPIFFE ID for an identity.
```

Comment:
Should `Workload` prepend `Identity` here so we don't have:
`Spiffe<IDentity><Identity>` feel extra strange?

Reply:
done!